src/rgw/store/dbstore/common/dbstore.h

```diff
@@ -87,9 +87,6 @@ struct DBOpObjectInfo {
   std::string min_marker;
   std::string max_marker;
   std::list<rgw_bucket_dir_entry> list_entries;
```

Comment:
this std::list is probably in the interface of rgw_rados or even cls;  we should revise this at some point (std::vector, most likely)

Reply:
okay thanks!.. have added it to code comment. will address it as part of generic dbstore cleanup later.